### PR TITLE
[risk=no] IAP test on aou-db-tst

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Testing identity aware proxy on aou-db-test

### CODE REVIEW

File: public-ui/app.yaml [hunk 1]
```diff
@@ -9,16 +9,23 @@ handlers:
   secure: always
   # TODO(calbach): Bump max-age to 1y once we've verified this is correct.
   http_headers:
+    Cache-Control: private
+    Access-Control-Allow-Credentials: true
```
Comment (SrushtiGangireddy) [inline public-ui/app.yaml]: Let me know if you want me to remove this too.

Comment (calbach) [inline public-ui/app.yaml]: Were these needed ultimately?

File: public-api/src/main/java/org/pmiops/workbench/config/WebSecurityConfig.java [hunk 2]
```diff
@@ -13,6 +14,7 @@
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
+        http.cors().configurationSource(request -> new CorsConfiguration().applyPermitDefaultValues());
```
Comment (calbach) [inline public-api/src/main/java/org/pmiops/workbench/config/WebSecurityConfig.java]: I'm guessing this one shoudl no longer be needed

**Comment by SrushtiGangireddy** [comment]
Closing this out as it is messed up due to rebase.

**Pull request closed without merging**